src/envo/envo-edit.owl

```diff
@@ -39742,6 +39745,51 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105008> (plastic fragment)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105008> "A piece of plastic which 1) is hard and 2) has an irregular or broken shape that implies it's being broken down from a large piece of litter."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105008> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105008> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105008> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105008> "2021-05-04T14:37:37Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105008> "Plastikfragment"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105008> "flake"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105008> "fragmento de plástico"@es)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105008> "frammento di plastica"@it)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105008> "granule"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105008> "envoPlastics"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105008> "plastic fragment"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105008> <http://purl.obolibrary.org/obo/ENVO_01000776>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105009> (plastic line)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105009> "A piece of plastic which 1) is long and fibrous and 2) has a length substantially longer than its width."@en)
```

Comment:
>A piece of plastic which 1) is long and fibrous and 2) has a length substantially longer than its width.

Perhaps slightly refine to:
>A piece of plastic which is linear and fibrous

We have `linear` from PATO, I believe. Also, does it have to be fibrous? Some fishing lines are monofilaments, not sure if that counts as fibrous.

Reply:
@pbuttigieg

Here is the original description from GESAMP:

> Long fibrous material that has a length substantially longer than its width.

and in later chapter Figure 9.4 point out `plastic line` as flexible and 1D.

Somehow I associated fibrous with flexible earlier but now I realised they aren't. So with your example of monofilaments, I think it make sense to leave fibrous out and maybe include the flexible part. Here is the definition I captured now:

> A piece of plastic which is linear and flexible.

and we have `linear` and `flexible` from PATO.